src/app/helptext/system/ssh-connections.ts

```diff
@@ -0,0 +1,38 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    setup_method_placeholder: T('Setup Method'),
+    setup_method_tooltip: T(''),
+
+    host_placeholder: T('Host'),
+    host_tooltip: T(''),
+
+    port_placeholder: T('Port'),
+    port_tooltip: T(''),
+
+    username_placeholder: T('Username'),
+    username_tooltip: T(''),
+
+    private_key_placeholder: T('Private Key'),
+    private_key_tooltip: T(''),
+
+    remote_host_key_placeholder: T('Remote Host Key'),
+    remote_host_key_tooltip: T(''),
+
+    cipher_placeholder: T('Cipher'),
+    cipher_tooltip: T(''),
+
+    connect_timeout_placeholder: T('Connect Timeout'),
+    connect_timeout_tooltip: T(''),
```

Comment:
from ssh_config(5):

Time to continuously attempt connecting to the remote system. Enter a number of seconds.

Reply:
This can mislead people. Nothing in our system "continuously attempts connecting", it is one `connect` call and that's it. I don't think we can and should explain what is a "connect timeout" in this tooltip, instead, just say "in seconds" or, better, have a widget with placeholder and remove tooltip completely.